vibrato/src/trainer/feature_rewriter.rs

```diff
@@ -43,7 +45,7 @@ impl FeatureRewriterBuilder {
     }
 
     #[allow(unused)]
-    pub fn add_rule<S>(&mut self, pattern: &[S], rewrite: &[S]) -> Result<()>
+    pub fn add_rule<S>(&mut self, pattern: &[S], rewrite: &[S])
```

Comment:
```suggestion
    /// Adds the rewrite rule associated with the pattern.
    /// If the pattern is shorter than the rewrite rule,
    /// the remainings are automatically padded with "*".
    pub fn add_rule<S>(&mut self, pattern: &[S], rewrite: &[S])
```
Could you add a comment on the new handling?